src/main/resources/data/exnihilosequentia/recipes/sieve/ens_dirt.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "exnihilosequentia:sieve",
+  "rolls": [
+    {
+      "chance": 1.0,
+      "mesh": "string"
+    },
+	{
+      "chance": 0.4,
+      "mesh": "string"
+    }
```

Comment:
Do you intend that it might drop twice?

Reply:
not really, but you can always get the dirt with a hoe. thats up to you if you want to drop it twice. its the same with gravel. i just know that you need 2 gravel and 2 dirt to create 4 coarse dirt. but it's up to you if you want drop chances.